i6_models/parts/blstm.py

```diff
@@ -0,0 +1,76 @@
+from dataclasses import dataclass
+import torch
+from torch import nn
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass()
+class BlstmEncoderConfig(ModelConfiguration):
+    num_layers: int
+    """number of bi-directional LSTM layers, minimum 2"""
+    input_dimension: int
```

Comment:
We have this somewhat inconsistent throughout other parts in the code. E.g. in `ConformerPositionwiseFeedForwardV1Config` we have `input_dim` and `hidden_dim`. Shouldn't we try to make it consistent?